flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -152,16 +137,15 @@
  * <p>NOTE: Please ensure to close and dispose any created keyed state backend in tests.
  */
 @SuppressWarnings("serial")
+@ExtendWith(ParameterizedTestExtension.class)
 public abstract class StateBackendTestBase<B extends AbstractStateBackend> extends TestLogger {
 
-    @Rule public final ExpectedException expectedException = ExpectedException.none();
-
-    @Before
+    @BeforeEach
     public void before() throws Exception {
         env = buildMockEnv();
     }
 
-    @After
+    @AfterEach
     public void after() {
```

Comment:
```suggestion
    void after() {
```
could be package private